Source/Logging/Internal/InternalLogger.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dolittle.Logging.Internal
+{
+    /// <summary>
+    /// Represents an implementation of <see cref="ILogger"/>.
+    /// </summary>
+    public abstract class InternalLogger : ILogger
```

Comment:
There are no abstract methods - we can therefor just keep this a regular class.
```suggestion
    public class InternalLogger : ILogger
```

Reply:
This class is not meant to be instanciated directly - as the InternalLogger<T> types will be created. But since we cant have collections of open generics, this was my solution to prevent myself from using the class.

Another option could be to make the constructor protected - would that be better?